packages/mantine/src/components/table/__tests__/Table.spec.tsx

```diff
@@ -508,5 +508,78 @@ describe('Table', () => {
             await user.click(screen.getByRole('button', {name: /2 selected/i}));
             expect(screen.queryAllByRole('row', {selected: true})).toEqual([]);
         });
+
+        it('prevents row selection if disableRowSelection is true', async () => {
+            const user = userEvent.setup({delay: null});
+
+            render(
+                <div>
+                    <div>I'm a header</div>
+                    <Table
+                        getRowId={({id}) => id}
+                        data={[
+                            {id: '🆔-1', firstName: 'first', lastName: 'last'},
+                            {id: '🆔-2', firstName: 'patate', lastName: 'king'},
+                        ]}
+                        columns={columns}
+                        multiRowSelectionEnabled
+                        disableRowSelection
+                    />
+                </div>
+            );
```

Comment:
Do you really need a header for your test?

```suggestion
            render(
                <Table
                    getRowId={({id}) => id}
                    data={[
                        {id: '🆔-1', firstName: 'first', lastName: 'last'},
                        {id: '🆔-2', firstName: 'patate', lastName: 'king'},
                    ]}
                    columns={columns}
                    multiRowSelectionEnabled
                    disableRowSelection
                />
            );
```

Reply:
No, I will remove it